students/grzegorz_bajorski/lesson_06/logger_wraper.py

```diff
@@ -0,0 +1,8 @@
+def inspect_args(*args, **kwargs):
```

Comment:
This solution is not compliance with the task. You should write a function, for instance,```logger_wrapper``` which will call ```inspect_args``` and then call another function (foo) that will be passed as a first argument to ```logeer_wrapper()```

```
def logger_wrapper(some_foo, *args, **kwargs ):
      inspect_args(*args, **kwargs)
      some_foo(*args, **kwargs)
```